CHANGELOG.md

```diff
@@ -341,13 +351,18 @@ Added
 - Added `set_path` option for salt-ssh shim. (#56627)
 - Added `win_wua.installed` to check a list of updates that apply to the current Windows build. (#56640)
 - Added ability to compare package versions in Jinja templates. (#56678)
+- Add function 'shutdown_host' to vmware cloud (#56711)
+- Add a new drbd monitor function via drbdadm to deprecate the old method of drbd-overview (#56768)
+- New requisite form: `onfail_all` (#47015)
 - Add `auto_detect` feature for `ssh_ext_alternatives`. (#56894)
 - Add ability to display sys.doc style outputs but without actually loading the module. (#56902)
 - Added plist serializer. (#56954)
 - Added support for onedir/pop-build Salt in the `pip` module. (#56988)
 - Add support for disks volumes in virt.running state (#57005)
 - Add virt.all_capabilities helper function (#57009)
 - supervisord.status_bool method (#57049)
+- Uninstall msi-installed salt (#57068)
```

Comment:
Listed as 56883 - this is probably the correct number, but that has the better description